### ISSUE TITLE
Unmatched dropout ratio in AIDO.RNA

### ISSUE THREAD

**Issue by ZhiyuanChen** [genbio-ai/AIDO#1]
In table 13 of the AIDO.RNA paper, the `attention dropout` and `residual dropout` is 0.1. 

In the [configuration file](https://huggingface.co/genbio-ai/AIDO.RNA-1.6B/edit/main/config.json), the `attention dropout` is set to 0. And in the [code file](https://github.com/genbio-ai/ModelGenerator/blob/main/modelgenerator/huggingface_models/rnabert/modeling_rnabert.py), no `residual dropout` is applied.

**Comment by LittletreeZou** [comment]
Our model was pre-trained using Megatron-LM and then converted to a Huggingface model for downstream evaluation. In downstream tasks, we did not need to use residual dropout, so it is not included in the Huggingface version.
